src/types/index.ts

```diff
@@ -16,3 +16,17 @@ interface GetLayout {
 export type PageWithLayout<P = Record<string, unknown>> = NextPage<P> & {
     getLayout: GetLayout;
 };
+
+export interface BookedEvent {
```

Comment:
Is this different in any way from `ZetkinEvent`? If there's very little difference, consider using the same type and just changing it when necessary.